keyboards/ergodox_ez/keymaps/rishka/default.png.md

```diff
@@ -0,0 +1 @@
+https://i.imgur.com/fKX0Zbs.png
```

Comment:
I think you want to have a proper readme.md which includes these images, using the `![](url)` syntax :)

Reply:
This was left over from the copy keyboard command, Ill just get rid of it